### DISCUSSION TITLE
data-validate dosent work proprely

### DISCUSSION THREAD

**Discussion by AZ77-Git** [olton/Metro-UI-CSS#1819] [Q&A]
Hello,

I have experienced an issue with data-validate form in metroui.

when i use data-role="validator", the default html validation like "required" or "type=email" dosent work anymore => view codepen https://codepen.io/AZ77/pen/poPYBNg
for exemple i am forced to use data-validate="required minlength=1" to simulate the html required function.

You can see the same form here https://codepen.io/AZ77/pen/dyWrrNN without the data-role validator and the default html validation works normaly like "type=email" and "required"

according to you how can I use data-role="validator" functions and in the same time keep options like required and input type email working.

thank you.

**Comment by AZ77-Git** [comment]
Hello,

Have you any idea ? 

Thanks.

**Comment by olton** [comment]
you must use attribute `data-validate="required"`
Have you read the documentation? https://metroui.org.ua/validator.html

**Comment by AZ77-Git** [comment]
Hello,

Yes I've read this documentation, but i've missed this information Sorry.

with data-validate="required", the invalid_feedback classe is used and shown as expected.

but if I fill the email field and enter something like this.is.not.an.email the invalid_feedback classe is not used and shown as expected.

remember the codpen link using data-validate classe https://codepen.io/AZ77/pen/poPYBNg

the link without data-validate and with an email field validatig correctly if the text is realy an email : https://codepen.io/AZ77/pen/dyWrrNN

Thank you again for your help.
